### PR TITLE
[Dashboard] Fix resource names formatted to google style + change the actions order + tooltips

### PR DESCRIPTION
**Pull request by rkooo567**
Signed-off-by: SangBin Cho <rkooo567@gmail.com>

## Why are these changes needed?

This PR does 4 things.
1. For actor table's required_resources column, we join entries by ","
2. Move Actions column next to IP (node table) and state (actor table) for a bette feature discoverability
3. Fix a bug that required_resources were formatted by google style (e.g., instead of `a_bc: 1`, it was formatted as `aBc: 1`).
4. Add tooltips to actor table columns

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/client/src/components/ActorTable.tsx [hunk 1]
```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
...
 
+  const columns = [
+    { label: "" },
+    { label: "ID" },
+    {
+      label: "Class",
+      helpInfo: (
+        <Typography>
+          The class name of the actor. For example, the below actor has a class
+          name `Actor`.
```
Comment (alanwguo) [inline dashboard/client/src/components/ActorTable.tsx:54]:
```suggestion
          name "Actor".
```

File: dashboard/client/src/components/ActorTable.tsx [hunk 2]
```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
...
+          <br />
+        </Typography>
+      ),
+    },
+    {
+      label: "Name",
+      helpInfo: (
+        <Typography>
+          The name of the actor given by an `name` argument. For example, this
+          actor's name is `unique_name`.
```
Comment (alanwguo) [inline dashboard/client/src/components/ActorTable.tsx:71]:
```suggestion
          The name of the actor given by the "name" argument. For example, this
          actor's name is "unique_name".
```

File: dashboard/client/src/components/ActorTable.tsx [hunk 3]
```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
...
+          Actor.options(name="unique_name").remote()
+        </Typography>
+      ),
+    },
+    {
+      label: "State",
+      helpInfo: (
+        <Typography>
+          The state of the actor. States are documented at a `ActorState` from a
+          `gcs.proto` file.
```
Comment (alanwguo) [inline dashboard/client/src/components/ActorTable.tsx:83]:
```suggestion
          The state of the actor. States are documented as "ActorState" in the
          "gcs.proto" file.
```

File: dashboard/client/src/components/ActorTable.tsx [hunk 4]
```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
...
+      helpInfo: (
+        <Typography>
+          A list of actions performable on this actor.
+          <br />
+          - Log: See a log message of this actor. Only available if a node is
+          alive.
+          <br />
+          - Stack Trace: Get a stacktrace of the alive actor.
+          <br />- Flame Graph: Get a flamegraph of the alive actor. It profiles
+          for 5 seconds by default.
```
Comment (alanwguo) [inline dashboard/client/src/components/ActorTable.tsx:98]:
```suggestion
          <br />- Flame Graph: Get a flamegraph for the next 5 seconds of an alive actor.
```

**Review by alanwguo — APPROVED** [review]

File: dashboard/client/src/components/ActorTable.tsx (hunk 3 above)
Comment (scottsun94) [reply dashboard/client/src/components/ActorTable.tsx:83]: @rkooo567 @alanwguo 
Shall we actually document the "states" of actors, nodes, etc. in the ray documentation? Probably in the documentation of dashboard or state api later? Maybe in 2.3 or + after we finalize the list of states.

File: dashboard/client/src/components/ActorTable.tsx [hunk 5]
```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
...
+        </Typography>
+      ),
+    },
+    {
+      label: "Actions",
+      helpInfo: (
+        <Typography>
+          A list of actions performable on this actor.
+          <br />
+          - Log: See a log message of this actor. Only available if a node is
```
Comment (scottsun94) [inline dashboard/client/src/components/ActorTable.tsx:93]: "Log: view log messages of this actor"?

**Comment by rkooo567** [comment]
We may need to cherry pick this change.

File: dashboard/client/src/components/ActorTable.tsx (hunk 3 above)
Comment (rkooo567) [reply dashboard/client/src/components/ActorTable.tsx]: Yeah sgtm cc @rickyyx  to record the work items!

**Pull request merged**